README.md

```diff
@@ -201,32 +201,34 @@ To ask the questions or discuss the library you can join our [telegram chat](htt
 
 ## Resources
 
-- [Forecasting with ETNA: Fast and Furious](https://medium.com/its-tinkoff/forecasting-with-etna-fast-and-furious-1b58e1453809) on Medium
```

Comment:
What is the logic behind such ordering?

Reply:
I tried to order this entries by chronological order.

Reply:
What do you think about this section in `README`? Should we keep it?

Reply:
I think that the importance and usefulness is better for ordering. As a rule people click only first links

Reply:
We should add resources with 2.0 api on top at least, I think.